systemtest/src/test/java/io/strimzi/systemtest/AllNamespaceST.java

```diff
@@ -86,13 +90,54 @@ void testUOWatchingOtherNamespace() {
         setNamespace(previousNamespace);
     }
 
+    @Test
+    void testUserInDifferentNamespace() throws Exception {
+        String startingNamespace = setNamespace(SECOND_NAMESPACE);
+        secondNamespaceResources.tlsUser(CLUSTER_NAME, USER_NAME).done();
+
+        StUtils.waitForSecretReady(USER_NAME);
+        Condition kafkaCondition = secondNamespaceResources.kafkaUser().inNamespace(SECOND_NAMESPACE).withName(USER_NAME).get()
+                .getStatus().getConditions().get(0);
+        LOGGER.info("Kafka User condition status: {}", kafkaCondition.getStatus());
+        LOGGER.info("Kafka User condition type: {}", kafkaCondition.getType());
+
+        assertEquals("Ready", kafkaCondition.getType());
+
+        List<Secret> secretsOfSecondNamespace = kubeClient(SECOND_NAMESPACE).listSecrets();
+
+        for (Secret s : secretsOfSecondNamespace) {
+            if (s.getMetadata().getName().equals(USER_NAME)) {
+                LOGGER.info("Copying secret {} from namespace {} to namespace {}", s, SECOND_NAMESPACE, THIRD_NAMESPACE);
+                copySecret(s, THIRD_NAMESPACE, USER_NAME);
+            }
+        }
+        waitForClusterAvailabilityTls(USER_NAME, THIRD_NAMESPACE);
+
+        setNamespace(startingNamespace);
+    }
+
+    void copySecret(Secret sourceSecret, String targetNamespace, String targetName) {
+        Secret s = new SecretBuilder(sourceSecret)
+                    .withNewMetadata()
+                        .withName(targetName)
+                        .withNamespace(targetNamespace)
+                    .endMetadata()
+                    .build();
+        kubeClient(targetNamespace).getClient().secrets().inNamespace(targetNamespace).createOrReplace(s);
+    }
+
     private void deployTestSpecificResources() {
+
         thirdNamespaceResources.kafkaEphemeral(CLUSTER_NAME, 1)
             .editSpec()
                 .editKafka()
-                    .withNewListeners()
-                        .withNewTls()
-                        .endTls()
+                    .editListeners()
+                        .withNewKafkaListenerExternalNodePort()
+                            .withNewOverrides()
```

Comment:
I think we don't need `override` bootstrap in that case, or do we?

Reply:
Yes, that's right. We don't need it.